charts/catalog/Chart.yaml

```diff
@@ -1,3 +1,4 @@
+apiVersion: v1
 name: catalog
 description: service-catalog webhook server and controller-manager helm chart
-version: 0.3.0
+version: 0.3.1
```

Comment:
please revert that change, as this is only bumped when we are doing a new Service Catalog release with a bunch and not when we are merging separate features to master

Reply:
@mszostok This is a kind of against best practices, isn't it? Charts should be immutable artifacts. As an example, someone who already downloaded/cached 0.3.0 and need priorityClass support might get unexpected results. Why not use appVersion?

Reply:
@mszostok ping

Reply:
hi, but the charts from the master are not pushed to any repository. 

Only if the service catalog is released then we push a new chart to the helm repo. Currently, we are using `version` to reflect the service catalog release version. I'm not against to change that approach and be compliant with best practices. But let's split that into separate tasks:
- merge that PR
- raise an issue and change the way how we treat the version and appVersion - gives other members to agree on it or reject :) If accepted then change the documentation and release process.

WDYT?

Reply:
@mszostok make sense, but it will not solve our current problem - we need 0.3.0 version with PriorityClasses support :)

Reply:
but it is not possible to update 0.3.0. The only way is to release 0.3.1

/cc @jberkhahn

Reply:
yeah, changing that number won't actually do anything. there's no way under our current workflow to backport functionality to a previous release.

Reply:
@mszostok I reverted chart version